### DISCUSSION TITLE
Login com qrcode parou de funcionar essa semana, ocorreu algo semelhante com alguém ?

### DISCUSSION THREAD

**Discussion by MauricioRFilho** [andreroggeri/pynubank#441] [Q&A]
Estava trabalhando com um app fazendo o login com qrcode e essa semana parou de funcionar...

**Comment by andreroggeri** [comment]
Você consegue logar pelo site normalmente?

**Reply by MauricioRFilho** [reply]
Pior que testei agora e funcionou no navegador, vou verificar novamente as requisições, ver se deixei passar algo.
